examples/msal-go/main.go

```diff
@@ -12,10 +12,13 @@ import (
 )
 
 func main() {
-	keyvaultName := os.Getenv("KEYVAULT_NAME")
-	secretName := os.Getenv("SECRET_NAME")
-
-	keyvaultURL := fmt.Sprintf("https://%s.vault.azure.net/", keyvaultName)
+	keyvaultURL := os.Getenv("KEYVAULT_URL")
+        if keyvaultURL == "" {
+                keyvaultName := os.Getenv("KEYVAULT_NAME")
```

Comment:
Could you move `keyvaultName` outside the if loop?

Reply:
build fixed. ptal

Reply:
> I suppose we can remove keyvaultName in future after the doc is changed to use keyvaultURL?

For this change, we should update the env var used across all the language examples here: https://github.com/Azure/azure-workload-identity/tree/main/examples